crates/astria-sequencer/src/authority/action.rs

```diff
@@ -74,6 +74,12 @@ impl ActionHandler for tendermint::validator::Update {
 
 #[async_trait::async_trait]
 impl ActionHandler for SudoAddressChangeAction {
+    async fn check_stateless(&self) -> Result<()> {
+        crate::address::ensure_base_prefix(&self.new_address)
+            .context("desired new sudo address is incorrect")?;
```

Comment:
perhaps "desired new sudo address has incorrect prefix"?

Reply:
Clarified the error in:  1634af